amida-tech/orange-api
# hotfix/dont-authenticate-options-requests

## hotfix/signup-login-email-tolower
Before this PR, orange-api tries to authenticate (controllers/helpers/auth.js) OPTIONS requests.
This is a problem because OPTIONS requests from certain places, such as the old orange clinical web portal, don't put the JWT in the authorization header.

This change makes it so that OPTIONS aren't forced to authenticate.

## Branch Setup
N/A

## Related Tickets

* [DEVOPS-471](https://jira.amida-tech.com/browse/ORANGE-471)

## Screenshots
N/A

## Testing Steps
Make an unauthenticated OPTIONS request to some orange api endpoint (perhaps `/v1/patients?limit=10&offset=0`) using something such as postman. You should NOT get a 401 back. You should get a 200 back with a response body something like "POST,GET,HEAD"

## Readiness
1. [ N/A ] This PR has full test coverage
2. [ N/A ] This PR passess all tests
3. [ Yes ] This PR has no linting errors
4. [ Yes ] This PR has been updated to include all changes from develop
5. [ N/A - tested via orange clinical web portal ] This PR has been tested in iOS *and* Android
6. [ N/A ] This PR's changes to configuration files for an instance's runtime have been documented
7. [ N/A ] This PR's required changes outside of the scope of this repository have been documented in this pull request
